bootcamp/tutorials/quickstart/hybrid_search_with_milvus.ipynb

```diff
@@ -0,0 +1,552 @@
+{
```

Comment:
<h1>Hybrid Search with Dense and Sparse Vectors in Milvus</h1><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>